Basic platforms fix (#183)
cylc/cylc-doc: src/user-guide/writing-suites/runtime.rst

Remote Task Hosting
-------------------

If a task declares a different platform to the one running the workflow,
Cylc will use non-interactive ssh to execute the task using the 

> :term:`job runner` and one of the hosts from the platform definition

Reviewer: ```suggestion
:term:`job runner` and one of the hosts from the :term:`platform` definition
```

> (platforms are defined in the ``global.cylc`` file at site level).

Reviewer: ```suggestion
(platforms are defined in ``global.cylc[platforms]``).
```

> If a task declares a platform other than the platform running the workflow,
> Cylc will use non-interactive ssh to execute the task using the 
> :term:`job runner`: and one of the hosts from the platform definition
> (platforms are defined in the ``global.cylc`` file at site level).

Reviewer 2: ```suggestion
If a task declares a different platform to the one running the workflow,
Cylc will use non-interactive ssh to execute the task using the 
:term:`job runner` and one of the hosts from the platform definition
(platforms are defined in the ``global.cylc`` file at site level).
```

For example:

.. code-block:: cylc

   [runtime]
       [[foo]]
           platform = orca

For this to work:

> - Non-interactive ssh is required from the platform running the workflow

Reviewer: ```suggestion
- Non-interactive ssh is required from the :term:`scheduler` host
```

> - Non-interactive ssh is required from the workflow platform to the remote
>   platform's hosts.

Reviewer 2: ```suggestion
- Non-interactive ssh is required from the platform running the workflow
  to the remote platform's hosts.
```

- Cylc must be installed on the hosts of the destination platform.

  - If polling task communication is used, there is no other
    requirement.
  - If SSH task communication is configured, non-interactive ssh is
    required from the task platform to the workflow platform.

>   - If (default) task communication is configured, the task platfom

Reviewer: ```suggestion
  - If TCP (default) task communication is configured, the task platform
```
